Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -216,6 +216,15 @@ class PruneCoreSymbolicationdCacheIfTooLarge(shell.ShellCommand):
                "/System/Library/Caches/com.apple.coresymbolicationd"]
 
 
+class PruneCoreSymbolicationdDataIfTooLarge(shell.ShellCommand):
+    name = "prune-coresymbolicationd-data-if-too-large"
+    description = ["pruning coresymbolicationd cache to < 10GB"]
+    descriptionDone = ["pruned coresymbolicationd cache"]
+    flunkOnFailure = False
+    haltOnFailure = False
+    command = ["python3", "Tools/Scripts/delete-if-too-large",
+               "/System/Volumes/Data/System/Library/Caches/com.apple.coresymbolicationd"]
```

Comment:
Isn't `/System/Volumes/Data/System/Library/Caches` just a worse way to write `/System/Library/Caches`?

Reply:
I believe it is. Though, for some reason we are seeing both of these paths needed.

Reply:
I think that's the same directory, just linked to two places in the hierarchy in some way.

Reply:
There's some machines in the fleet that seem to need this path for some reason.